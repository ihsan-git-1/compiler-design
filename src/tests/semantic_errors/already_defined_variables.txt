class test {
    int a(){
        int b= 5;
        double b=5.6;
    }
}