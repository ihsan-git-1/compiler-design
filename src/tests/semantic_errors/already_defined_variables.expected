The name b already defined (4,16)
